governance-charter.md

```diff
@@ -0,0 +1,234 @@
+election activities while they serve as continuity members. Members of the
+bootstrap committee will refrain from endorsing or otherwise advocating for any
+candidate (with the exception that the members of the bootstrap committee may
+vote in the elections, and may choose to run in the 2021 election).
+
+### Members of Standing
+
+Standing in the OpenTelemetry community will be defined by the union of:
+
+- Active members - those with 20 contributions (PRs, Issues, Comments etc.) in
+  the prior rolling year.
+- All approvers, and maintainers in the OpenTelemetry organization
+
+Members who don’t meet the criteria above and are part of the following are
+encouraged to help us evolve the definition of Member of Standing through the
+Exception Process outline below.
+
+- Maintainers for projects (vendors, OSS observability solutions, etc) with
+  strong end-user adoption that have taken a hard dependency on OpenTelemetry
+  APIs or data formats (one representative per project)
+- Representatives from major organizations that have adopted OpenTelemetry as
+  end-users (one representative per organization)
+
+The bootstrap committee explicitly believes that this heuristic will be
+inaccurate and not represent the entire community. An exception form will be
+provided for people who have contributed to the project but may not meet these
+criteria. Exception eligibility applications will be approved by the bootstrap
+committee by a simple majority vote. The exception process will be used as data
+points for refining the criteria for the future.
+
+Within its first year, the Steering Committee will redefine these criteria
+around a more robust idea of community membership.
+
+### Eligibility for candidacy
+
+Anyone may nominate either themselves or someone else to be a candidate in the
+election. To be ratified as a candidate, the nominee must accept the nomination
+and three Members of Standing (including the nominator, if she/he has standing)
+from three different employers, must endorse the nomination.
+
+Nominators are free to nominate as many people as they wish to. Members of
+Standing may endorse multiple nominees, but we expect endorsements to be in good
+faith. If this turns out to be a problem, this will be reconsidered.
+
+### Eligibility for voting
+
+All Members of Standing are eligible to vote for the governance committee
+members. 
+
+### Election process
+
+Elections will be held using time-limited Condorcet ranking on
+[CIVS](http://civs.cs.cornell.edu/). The top vote getters will be elected to the
+respective positions. 
+
+### Maximal representation
+
+To encourage diversity there will be a maximum of one-third representation on
+the Steering Committee from any one company at any time. If the outcomes of an
+election result in greater than 1/3 representation (or maximum of two, whichever
+is greater), the lowest vote getters from any particular company will be removed
+until representation on the committee is less than one-third.
```

Comment:
It is also worth saying who gets the empty seat after removal (I am guessing the person with the next highest vote that meets maximal representation rule, but would be good to mention that explicitly).

Reply:
there may be a need for a special election.

Reply:
yes, I think this is assumed